### PR TITLE
fix(core/event-list-item): fix right padding

### PR DESCRIPTION
**Pull request by matthiashader**
## Pull request checklist

Please check if your PR fulfills the following requirements:

- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [x] Build (`yarn build`) was run locally and any changes were pushed
- [x] Unit tests (`yarn test`) were run locally and passed
- [x] Visual Regression Tests (`yarn visual-regression`) were run locally and passed
- [x] Linting (`npm lint`) was run locally and passed

## Pull request type

Please check the type of change your PR introduces:

- [x] Bug fix
- [ ] Feature
- [ ] Refactoring (no functional changes, no API changes)
- [ ] Build related changes
- [ ] Documentation content changes
- [ ] Other (please describe):

## What is the current behavior?

GitHub Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No

### CODE REVIEW

**Review by collins-k — COMMENTED** [review]
I personally prefer to use the `padding-inline` property as a shortcut for defining left and right padding

File: packages/core/src/tests/event-list/chevron/index.html
Comment (nuke-ellington) [inline packages/core/src/tests/event-list/chevron/index.html:1]: Unnecessary reformatting?

File: packages/core/src/components/event-list-item/event-list-item.scss [hunk 1]
```diff
@@ -82,6 +82,7 @@ $default-event-list-item-height: 2.5rem;
       max-height: var(--event-list-item-height, 100%);
       white-space: var(--event-list-item-content-white-space, inherit);
       overflow: hidden;
+      padding-right: $small-space;
```
Comment (nuke-ellington) [inline packages/core/src/components/event-list-item/event-list-item.scss:85]:
```suggestion
      padding-inline-end: $small-space;
```

Comment (nuke-ellington) [reply packages/core/src/components/event-list-item/event-list-item.scss:85]: Looks to me like this change does not have any effect on the rendered component.
Also font truncation is not working in event list item currently. Maybe we can fix this with this PR?

**Review by nuke-ellington — CHANGES_REQUESTED** [review]

File: packages/core/src/components/event-list-item/event-list-item.scss (hunk 1 above)
Comment (matthiashader) [reply packages/core/src/components/event-list-item/event-list-item.scss:85]: > padding-inline-end: $small-space;
With this change the icon will also be have the padding, i dont know if this is wanted.

File: packages/core/src/tests/event-list/chevron/index.html
Comment (matthiashader) [reply packages/core/src/tests/event-list/chevron/index.html:1]: for me it looked like something was wrong with the formatting, in the first place. e.g:
      <ix-event-list-item color="color-primary" disabled
        >Text 1</ix-event-list-item
      >
Is this intendent?

**Pull request merged**